phoebe/atmospheres/passbands.py

```diff
@@ -354,9 +373,9 @@ def save(self, archive, overwrite=True, update_timestamp=True, export_inorm_tabl
             have been deprecated since phoebe 2.4; for backwards compatibility
             we still may need to export them to fits files so that pre-2.4
             versions can use the same passband files.
-        * `history_entry` (string, optional): history entry to append to the
-            fits file.  Note that previous entries will be maintained if
-            (and only if) overwriting an existing file with `overwrite=True`.
+        * `export_legacy_comments` (bool, optional, default=True): whether to
+            export the dummy COMMENTS card (used in old passbands) in addition
+            to the new COMMENT card.
```

Comment:
is there a reason we can't just keep calling it `COMMENTS`?  Would that not be backwards-compatible either?

Reply:
yep, COMMENT is a reserved fits keyword that should be used exclusively because fits handles multiple instances of COMMENT natively under the hood. COMMENTS, on the other hand, is not standard and we need to implement the parsing logic in phoebe, which is a pain and it also violates fits standards. So I'd say we should stick to COMMENT. More on that [here](https://docs.astropy.org/en/stable/io/fits/index.html).